admin/test/test_cluster_setup.py

```diff
@@ -0,0 +1,25 @@
+import mock
+
+from twisted.python.usage import UsageError
+
+from flocker.testtools import TestCase
+
+from ..acceptance import CommonOptions
+from ..cluster_setup import RunOptions
+
+
+class RunOptionsTest(TestCase):
+
+    @mock.patch.object(CommonOptions, 'postOptions')
```

Comment:
That actually changes the nature of the test, as I'm mocking the postOptions of CommonOptions (the parent) rather than of RunOptions. This requires more code and would rapidly become ugly. I'm willing to change it here for the sake of getting this landed, but I'd question whether it's correct to avoid writing certain kinds of tests in order to avoid adding mock as a dependency.